SteamList/GamesViewController.swift

```diff
@@ -9,11 +9,40 @@ import UIKit
 
 class GamesViewController: UIViewController {
 
+    @IBOutlet weak var tableView: UITableView!
+    private let numberOfGames = 100
+    
     override func viewDidLoad() {
         super.viewDidLoad()
 
+        tableView.delegate = self
```

Comment:
It's better to take out the contextual logic of your tableView into a separate method rather than flood viewDidLoud. 
It would be better it you created `func configureTableView()` method

Reply:
done